relay-server/src/services/processor.rs

```diff
@@ -1893,6 +1895,34 @@ impl EnvelopeProcessorService {
                 .parametrize_dsc_transaction(&sampling_state.config.tx_name_rules);
         }
 
+        macro_rules! call {
+            ($fn_name:ident ( $($args:expr),* )) => {{
```

Comment:
This solution for params was the easiest i could think of. I am not fond of using macros but for this use case I feel like it's pretty okay.

Reply:
Isn't this macro the same as `run`, but also allows specifying arguments? I think the two macros can be unified.

Reply:
Yes, we could, I am currently discussing offline which approach we want to take.